lib/internal/streams/end-of-stream.js

```diff
@@ -74,13 +49,12 @@ function eos(stream, options, callback) {
   callback = once(callback);
 
   const readable = options.readable ||
-    (options.readable !== false && isReadable(stream));
+    (options.readable !== false && isReadableStream(stream));
```

Comment:
Let's please use `isStreamReadable()` as the pattern for these to avoid confusion with the `ReadableStream` impl.

Reply:
For me that's something totally different:

`isStreamReadable` means that it's a readable stream and `stream.readable === true`.
`isReadableStream` means that it's a readable stream.

Reply:
I was actually considering whether I should rename `isReadable` to `isStreamReadable`.

Reply:
I get that, and it is confusing. But internal/web/readablestream already uses and exports an isReadableStream method, so this would definitely cause some confusion.

Reply:
So what do we do? At least they live in different modules...? Is that sufficient?

Reply:
btw, this is not public api for now, only internal

Reply:
```js
const stream = require('stream')
const web = require('web')

stream.isReadableStream(obj)
web.isReadableStream(obj)
```